administration/src/cards/CardBlueprint.ts

```diff
@@ -69,6 +88,15 @@ export class CardBlueprint {
     return this.expirationDate !== null && this.expirationDate.isAfter(today)
   }
 
+  setExpirationDate = (value: string) => {
+    if (value.length === 0) return null
```

Comment:
```suggestion
    if (value.length === 0) return
```
i think this should be fine without null